### DISCUSSION TITLE
Welcome to MAgPIE Discussions!

### DISCUSSION THREAD

**Discussion by tscheypidi** [magpiemodel/magpie#236] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saxenasatyam** [comment]
Hi Jan,

I am stuck at an error while running a shiny app. Earlier it was working but not know. 
It is showing-:  Error: 'appMAgPIElocal' is not an exported object from 'namespace:shinyresults'. 
Can you please guide me with some solution for this issue?

**Reply by tscheypidi** [reply]
it is now called `appResultsLocal`. 

Best, Jan

**Reply by saxenasatyam** [reply]
Thanks for resolving this Jan.
I am facing a new problem while running shiny, It is picking up only 2 scenarios out of the 4 available scenarios. I am unable to crack this issue because there is no such error line, I am receiving.

**Reply by abhimishr** [reply]
Can you confirm if you have a file named `report.rds` in the output folders which don't show up in `appResultsLocal`?

**Reply by saxenasatyam** [reply]
Hi Abhi,

In the output folder, I am having summary.rds in place of report.rds

**Reply by abhimishr** [reply]
@tscheypidi we only need report.rds to go through the app yes?

@saxenasatyam Following will probably work on a recent-ish magpie version - Can you also share which model version you are using?
Meanwhile can you run `report_rds` output script on your model outputs?
1. Open command prompt from your main model folder
2. Rscript output.R
3. Select the runs which dont show up on appResults
4. select `rds_report` - wait until the process finishes.
5. load `appResultsLocal` again and see if it recognizes the runs for which you created `report.rds` via ``rds_report` from previous step

**Reply by tscheypidi** [reply]
@abhimishr I think so, yes
